### PR TITLE
Brightlystake endpoints

### PR DESCRIPTION
**Pull request by Brightlystake**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: nodes/mainnet.md [hunk 1]
```diff
@@ -99,20 +99,25 @@ RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with
 - `public-celestia-rpc.numia.xyz`
 - `celestia-rpc.mesa.newmetric.xyz`
 - `rpc.lunaroasis.net`
+- `https://celestia-rpc.brightlystake.com`
```
Comment (jcstein) [inline nodes/mainnet.md:102]:
```suggestion
- `celestia-rpc.brightlystake.com`
```

I'm removing "https://" as the CLI takes in a bare url

File: nodes/mainnet.md [hunk 2]
```diff
@@ -99,20 +99,25 @@ RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with
...
 
 ### API endpoints
 
 - `public-celestia-lcd.numia.xyz`
 - `celestia-rest.mesa.newmetric.xyz`
 - `api.celestia.pops.one`
 - `api.lunaroasis.net`
+- `https://celestia-rpc.brightlystake.com/api`
```
Comment (jcstein) [inline nodes/mainnet.md:110]:
```suggestion
- `celestia-rpc.brightlystake.com/api`
```

File: nodes/mainnet.md [hunk 3]
```diff
@@ -99,20 +99,25 @@ RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with
...
 
 ### gRPC endpoints
 
 - `public-celestia-grpc.numia.xyz`
 - `celestia-grpc.mesa.newmetric.xyz`
 - `grpc.celestia.pops.one`
 - `grpc.lunaroasis.net:443`
+- `celestia-rpc.brightlystake.com:9090`
+
+You can find the status of these endpoints [here](https://celestia-tools.brightlystake.com/)
```
Comment (jcstein) [inline nodes/mainnet.md:120]:
```suggestion
```

Comment (jcstein) [reply nodes/mainnet.md:120]: removing, as it is not for the status of all of "these" endpoints on the page, just brightlystake

**Review by jcstein — CHANGES_REQUESTED** [review]

File: nodes/mainnet.md (hunk 3 above)
Comment (Brightlystake) [reply nodes/mainnet.md:120]: Screenshot 2023-11-01 at 20 00 39
We have for all the rpc endpoints in the github bud, soon i will finish the work for grpc as well.

Comment (Brightlystake) [reply nodes/mainnet.md:120]: @jcstein

Comment (Brightlystake) [reply nodes/mainnet.md:120]: right now we have url fetched every hour automatically, rpc and grpc endpoints are active

Comment (jcstein) [reply nodes/mainnet.md:120]: "bud" okay...thanks for the context.

Comment (jcstein) [reply nodes/mainnet.md:120]: > We have for all the rpc endpoints in the github bud, soon i will finish the work for grpc as well.
got it. thanks bud...

it does not seem to show all endpoints, though, just the consensus endpoints. is this meant to be a dashboard to show [da rpc endpoints](https://docs.celestia.org/nodes/mainnet#da-rpc-endpoints) or [consensus rpc endpoints](https://docs.celestia.org/nodes/mainnet#consensus-rpc-endpoints)? it appears its showing just consensus for mainnet

Comment (jcstein) [reply nodes/mainnet.md:120]: > We have for all the rpc endpoints in the github bud, soon i will finish the work for grpc as well.
got it. thanks...

it does not seem to show all endpoints, though, just the consensus endpoints. is this meant to be a dashboard to show [da rpc endpoints](https://docs.celestia.org/nodes/mainnet#da-rpc-endpoints) or [consensus rpc endpoints](https://docs.celestia.org/nodes/mainnet#consensus-rpc-endpoints)? it appears its showing just consensus for mainnet, but is listed under gRPC endpoints

Comment (jcstein) [reply nodes/mainnet.md:120]: could you maybe show different columns with the "type" of endpoint perhaps?

File: nodes/mainnet.md [hunk 4]
```diff
@@ -108,14 +109,18 @@ RPCs for DA nodes to initialise or start your celestia-node to Mainnet Beta with
...
+- `celestia-rpc.brightlystake.com/api`
 
 ### gRPC endpoints
 
 - `public-celestia-grpc.numia.xyz`
 - `celestia-grpc.mesa.newmetric.xyz`
 - `grpc.celestia.pops.one`
 - `grpc.lunaroasis.net:443`
-- `grpc.celestia.nodestake.top`
+- - `grpc.celestia.nodestake.top`
```
Comment (jcstein) [inline nodes/mainnet.md:120]:
```suggestion
- `grpc.celestia.nodestake.top`
```

File: nodes/mainnet.md (hunk 3 above)
Comment (Staking7pc) [reply nodes/mainnet.md:123]: I have now made changes to 
- test for http and https
- with respect to the da rpc end point vs consensus 
we can identify with INDEXER = ON and EARLIEST BLOCK = 1 

i have added that as a note bud. Hope this is fine.

Comment (Staking7pc) [reply nodes/mainnet.md:123]: @jcstein please take a look once and correct me if my understanding is incorrect

Comment (jcstein) [reply nodes/mainnet.md:123]: LGTM!

**Review by jcstein — APPROVED** [review]

**Pull request merged**